kuma/users/templates/users/user_detail.html

```diff
@@ -1,20 +1,21 @@
 {% extends "base.html" %}
 
-{% set display_name = user_display(profile.user) %}
-{% set username  = profile.user.username %}
-{% set accounts = social_accounts(profile.user) %}
+{% set display_name = user_display(user) %}
+{% set username  = detail_user.username %}
+{% set accounts = social_accounts(user) %}
+{% set is_me = request.user.pk == detail_user.pk %}
```

Comment:
Woops, yeah, gonna double check again. `detail_user` is the user requested in this view, while `user` is the one added to the context by the auth context processor.